src/main/java/edu/hm/webtec/entities/Topic.java

```diff
@@ -0,0 +1,43 @@
+package edu.hm.webtec.entities;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+
+/**
+ * A topic describes what subjects a question covers. A question can have multiple topics, for example "java" and "math".
+ */
+@Entity
+public class Topic {
```

Comment:
Serializeable finde ich blöd, weil das markiert ja nur, dass man es im memory persistieren kann, und das darf man nicht weil der einzige konsistente stand in der DB liegt.